v4/source/system.yaml

```diff
@@ -578,6 +578,14 @@
         Must have `manage_system` permission.
 
         __Minimum server version__: 5.20
+
+        ##### Note
+
+        The Plugins are stored as a map, and since a map may recursively go 
+        down to any depth, individual fields of a map are not changed. 
+        We would want to avoid adding more complexity by trying to support 
+        patch on a map. We'd advise switching to the update config 
+        (PUT api/v4/config) endpoint as a workaround.
```

Comment:
```suggestion
        The Plugins are stored as a map, and since a map may recursively go 
        down to any depth, individual fields of a map are not changed. 
        Avoid adding more complexity by attempting to support 
        patch on a map by using the `update config` 
        (PUT api/v4/config) endpoint as a workaround.
```

Reply:
@hanzei - I've provided editorial input that requires a technical review to ensure that what's being conveyed is both accurate and clear.